src/attribute.rs

```diff
@@ -0,0 +1,787 @@
+use std::str::FromStr;
+
+use anyhow::{anyhow, Result};
+use strum_macros::{Display, EnumIter, EnumString};
+
+#[derive(Debug, PartialEq)]
+pub enum PacketType {
```

Comment:
The `PacketType` might be misleading in other ways. Instead, I suggest using `RawEventKind`, which also aligns with Giganto's terminology.

Reply:
done